test/.excludes-zjit/TestThread.rb

```diff
@@ -1,2 +0,0 @@
-exclude(:test_switch_while_busy_loop, 'Test hangs with ZJIT')
```

Comment:
This failed too https://github.com/ruby/ruby/actions/runs/16733873773/job/47368171297?pr=14097. 

When you file a PR that enables more test-all test cases next time, could you consider marking the PR ready for review after both test-all jobs (x86_64 and arm64) pass on the CI? It seems like the CI produces different results from your local environment.

Reply:
The other test failed too after enabling it. Disabled it again https://github.com/ruby/ruby/commit/196155812be09fdc4c36f8d7d6d1fbc69948765c.

Reply:
Apology and thanks for adding them back 🙏